src/utils/vm_mount.cpp

```diff
@@ -111,6 +113,36 @@ mp::VMMount::VMMount(const QJsonObject& json) : VMMount{parse_json(json)} // del
 {
 }
 
+void mp::VMMount::resolve_source_path()
```

Comment:
I was thinking about the same thing. This seems to be a private function to me.

Reply:
I think doing it in the constructor would be good, I'll look into it and any implications it has.

Reply:
It seems that doing it in the constructor makes it so that the source path is persisted as the place the symlink points to. This changes the behavior a bit so that symlinks are only ever resolved once instead of when the daemon restarts. It's still a fix for the issue but acts a bit different, what do you think?